examples/SubscribeHelloWorldBestEffort/README.md

```diff
@@ -0,0 +1,26 @@
+# SubscribeHelloWorldBestEffort example
+
+This example will show how to receive data from the DDS World creating a client subscriber in best effort mode..
+In order to compile this example, it is necessary to have the following profiles should be enabled:
+
+- `UCLIENT_PROFILE_UDP`
+
+## Usage
+1. Run an agent in port *2018*.
+2. Run the *PublisherHelloWorldBestEffort* example or some publisher that can send the *HelloWorld* topic.
+3. Run the *SubscriberHelloWorldBestEffort* example.
+   The example can be parameterized with the number of topics that will be listened.
```

Comment:
```suggestion
   The example expects first and second argument to be IP address and port where the Micro XRCE-DDS Agent is running. It can also be parameterized with the amount of times that the topic will be listened.
```